rollup/runtime/src/runtime_config.rs

```diff
@@ -1040,10 +837,12 @@ pub mod config {
 					let refund_imbalance = C::deposit_into_existing(token_id, &who, refund_amount)
 						.unwrap_or_else(|_| C::PositiveImbalance::from_zero(token_id));
 					// merge the imbalance caused by paying the fees and refunding parts of it again.
-					let adjusted_paid = paid
-						.offset(refund_imbalance)
-						.same()
-						.map_err(|_| TransactionValidityError::Invalid(InvalidTransaction::Payment))?;
+					let adjusted_paid = match paid.offset(refund_imbalance) {
+						SameOrOther::Same(a) => Ok(a),
+						SameOrOther::None => Ok(C::NegativeImbalance::from_zero(token_id)),
+						SameOrOther::Other(b) => Err(b),
+					}
```

Comment:
`SameOrOther::same()` in case of None used the `Default::default()`
added a PR to orml to remove the `Default` impl for imbalanced and replace with `fail!(...)`